### PR TITLE
chore(notes): update release notes for milestone

### PR DESCRIPTION
**Pull request by dgutride**
Adds release notes for 6/11 and 7/2 release bundles

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,3 +1,117 @@
...
+*   **Breadcrumb switcher:** Disabled autocomplete ([#2407](https://github.com/patternfly/patternfly-react/pull/2407))
+*   **Card:** Add compacted card variation ([#2399](https://github.com/patternfly/patternfly-react/pull/2399))
+*   **Checkbox:** Added 3rd state to checkbox controlled by consumer ([#2252](https://github.com/patternfly/patternfly-react/pull/2252))
+*   **Chip group:** Added heading level prop to toolbar chip group label ([#2278](https://github.com/patternfly/patternfly-react/pull/2278))
+*   **Data list:** Added DataListRow to imports ([#2353](https://github.com/patternfly/patternfly-react/pull/2353))
+*   **Dropdown:** Automatically focused first dropdown item when DropdownMenu ([#2355](https://github.com/patternfly/patternfly-react/pull/2355))
+*   **Expandable:** Added button type to the expandable component ([#2340](https://github.com/patternfly/patternfly-react/pull/2340))
+*   **Input group:** Convert input group to typescript ([#2220](https://github.com/patternfly/patternfly-react/pull/2220))
+*   **Label:** added typescript flag to the label docs ([#2343](https://github.com/patternfly/patternfly-react/pull/2343))
```
Comment (tlabaj) [inline RELEASE-NOTES.md]: This should be TypeScript?

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,3 +1,117 @@
...
+    *   Wrapped form label text in new element ([#2322](https://github.com/patternfly/patternfly-react/pull/2332))
+*   **Grid:**
+    *   Removed size props from {...props} ([#2404](https://github.com/patternfly/patternfly-react/pull/2404))
+    *   Updated grid layout to support new 2xl breakpoint ([#2305](https://github.com/patternfly/patternfly-react/pull/2305))
+*   **Tabs:**
+    *   Fixed typescript errors and revert `TabContainer` to a component ([#2402](https://github.com/patternfly/patternfly-react/pull/2402))
+    *   Remove comment from code ([#2351](https://github.com/patternfly/patternfly-react/pull/2351))
+    *   Made variant prop optional ([#2348](https://github.com/patternfly/patternfly-react/pull/2348))
+
+**Typescript conversion**
```
Comment (tlabaj) [inline RELEASE-NOTES.md]: Should be TypeScript and everywhere else.

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,3 +1,117 @@
...
+*   **Select:** Added keyboard interaction to typeahead variants, other variants remained unchanged. Enter, with select closed, toggles open the menu. Focus should now remains on input. Arrow up and down navigates the options with a focus styling. Enter, with select opened, selects the highlighted item. ([#2013](https://github.com/patternfly/patternfly-react/pull/2013))
+*   **Tabs:** fixed tabs ref proptype ([#2158](https://github.com/patternfly/patternfly-react/pull/2158))
+
+**Table (@patternfly/react-table@2.11.1):**
+*   Added React-Virtualized PF4 tables ([#2011](https://github.com/patternfly/patternfly-react/pull/2011))
+*   Exported RowWrapper implementation. When using react table in typescript project there was no way to import `RowWrapper` implementation. This PR fixed this problem by exporting `RowWrapperProps` as interface and `RowWrapper` as implementation. ([#2118](https://github.com/patternfly/patternfly-react/pull/2118))
```
Comment (tlabaj) [inline RELEASE-NOTES.md]: Should be TypeScript

File: RELEASE-NOTES.md (hunk 1 above)
Comment (dgutride) [reply RELEASE-NOTES.md]: @tlabaj  - I don't understand - these are component changes, not typescript conversions - wh not put them with component changes?

**Pull request merged**